api/src/types/database.ts

```diff
@@ -2,3 +2,15 @@ export type Driver = 'mysql' | 'pg' | 'cockroachdb' | 'sqlite3' | 'oracledb' | '
 
 export const DatabaseClients = ['mysql', 'postgres', 'cockroachdb', 'sqlite', 'oracle', 'mssql', 'redshift'] as const;
 export type DatabaseClient = (typeof DatabaseClients)[number];
+
+export const DatabaseVersionedClients = [
+	'postgres12',
+	'oracle12',
+	'mysql5',
+	'mysql8',
+	'mariadb',
+	'mssql13',
+	'sqlite',
+	'fallback',
+] as const;
```

Comment:
Not a fan of how tightly we're coupling the database vendors to the API codebase now..